app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -41,56 +39,72 @@ const Notes = () => {
 
   return (
     <>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            onClick={() => setShowNewNotePane(true)}
-            label="Add New Note"
-            icon="ri-add-line"
-          />
-        }
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            searchProps={{
-              value: searchTerm,
-              onChange: e => setSearchTerm(e.target.value),
-              clear: () => setSearchTerm("")
-            }}
-            deleteButtonProps={{
-              onClick: () => setShowDeleteAlert(true),
-              disabled: !selectedNoteIds.length
-            }}
+      <div className="flex w-full">
+        <Menubar />
+        <Container>
+          <Header
+            title="All Notes"
+            menuBarToggle={() => {}}
+            actionBlock={
+              <div className="flex ">
+                <div className="w-96">
+                  <Input
+                    className="pr-4 w-90"
```

Comment:
Please use `space-x-*` utils on the parent wrapper to create horizontal space between the input field and button.